olm/src/nativeTest/kotlin/io/github/matrixkt/olm/runTest.kt

```diff
@@ -0,0 +1,8 @@
+package io.github.matrixkt.olm
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.runBlocking
+
+actual fun BaseTest.runTest(block: () -> Unit) = runBlocking {
+    block()
+}
```

Comment:
No need for `runBlocking`.

```suggestion
actual fun BaseTest.runTest(block: () -> Unit) {
    block()
}
```